modules/ROOT/pages/using-policies.adoc

```diff
@@ -36,7 +36,7 @@ For details about the configuration parameters for each of the policies, see the
 
 == See Also
 
-* xref:client-id-based-policies.adoc[Reviewing Client ID-Based Policies concepts]
+* xref:client-id-based-policies.adoc[Reviewing Client ID Policies]
 * xref:gatekeeper.adoc[Reviewing API Gateway Gatekeeper Enhanced Security Reference]
-* xref:exchange::about-sharing-assets.adoc[About Sharing Assets, including an API]
-* xref:exchange::to-request-access.adoc[To Request Access to an API]
\ No newline at end of file
+* xref:exchange::about-sharing-assets.adoc[Sharing Assets(including an API)]
```

Comment:
Avoid using parentheses--or any punctuation--in titles. The clarification in the parens should be made in the text that follows the title, in much the same way that we might use an abbreviation in a title but then spell it in the text immediately following:
```suggestion
* xref:exchange::about-sharing-assets.adoc[Sharing Assets]
```
You *could* use the following, but it is misleading: do the assets include the API or is the API just one type of many assets?
```suggestion
* xref:exchange::about-sharing-assets.adoc[Sharing Assets Including an API]
```

Reply:
Done! Thanks!